lib/web/integrations.go

```diff
@@ -219,6 +223,79 @@ func (h *Handler) integrationsDelete(w http.ResponseWriter, r *http.Request, p h
 	return OK(), nil
 }
 
+// integrationsDelete removes an Integration based on its name
+func (h *Handler) integrationsDeleteWithCleanup(w http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (interface{}, error) {
+	values := r.URL.Query()
+	orgName := values.Get("orgName")
```

Comment:
This method is generic in that it could theoretically support any integration - GitHub just happens to be the first one supported.

`orgName` is a GitHub-specific field, but we're checking it here at the top level.

I wonder if a better approach would be to get the name and subkind first, then switch on the subkind and only start looking at GitHub-specific fields after we know we're dealing with a GitHub integration.